3648/2/Gemfile

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+gem 'redis'
```

Comment:
Every time lock gems that you use. Because if you don't lock them, there is a chance that gem will update with `bundle install` and will break the app